lib/vanagon/platform/windows.rb

```diff
@@ -1,6 +1,7 @@
 class Vanagon
   class Platform
     class Windows < Vanagon::Platform
+      attr_accessor :proj_root
```

Comment:
So is it acceptable to add root_dir to the Project initialisation with an accessor?
Then work up a mocked data structure for the purposes of this test ?